client/my-sites/plans-comparison/plans-comparison-features.tsx

```diff
@@ -120,7 +120,9 @@ export const planComparisonFeatures: PlanComparisonFeature[] = [
 
 			return feature
 				? translate( 'Custom domain name is free for one year!' )
-				: translate( 'Custom domain name is not included' );
+				: translate( 'Custom domain name is {{strong}}not{{/strong}} included', {
```

Comment:
@niranjan-uma-shankar will this create a new call for translations? Any thoughts to get around if so, as it's only stylistic changes? :-)